content/guides/models/app/features/panels/line-plot/_index.md

```diff
@@ -15,63 +15,62 @@ Line plots show up by default when you plot metrics over time with **wandb.log()
+      * **Relative Time (Process)** is time inside the running process, so if you started a run and ran for 10 seconds and resumed a day later that point would be plotted at 10s.
+      * **Wall Time** is minutes elapsed since the start of the first run on the graph.
+      * **Step** increments by default each time `wandb.log()` is called, and is supposed to reflect the number of training steps you've logged from your model.
+    * **Y**: Select one or more y-axes from the logged values, including metrics and hyperparameters that change over time.
+    * **X Axis** and **Y Axis** minimum and maximum values (optional).
+    * **Point aggregation method**. Either **Random sampling** (the default) or **Full fidelity**. Refer to [Sampling]({{< relref "sampling.md" >}}).
+    * **Smoothing**: Change the smoothing on the line plot. Defaults to **Time weighted EMA**. Other values include **No smoothing**, **Running average**, and **Gaussian**.
+    * **Outliers**: Rescale to exclude outliers from the default plot min and max scale.
+    * **Max number of runs or groups**: Show more lines on the line plot at once by increasing this number, which defaults to 10 runs. You'll see the message "Showing first 10 runs" on the top of the chart if there are more than 10 runs available but the chart is constraining the number visible.
+    * **Chart type**: Change between a line plot, an area plot, and a percentage area plot.
+  - **Grouping**: Settings for whether and how to group and aggregate runs.
+    * **Group by**: Select a column, and all the runs with the same value in that column will be grouped together.
+    * **Agg**: Aggregation— the value of the line on the graph. The options are mean, median, min, and max of the group.
+  - **Chart**: Titles for the panel, the X-axis, and Ythe -axis, whether or not to show the legend and its position, and font settings for the plot.
+  - **Legend**: Customize the appearance of the panel's legend:
+    * **Legend**: The field in the legend for each line in the plot in the legend of the plot for each line.
+    * **Legend template**: Define a fully customizable template for the legend, specifying exactly what text and variables you want to show up in the template at the top of the line plot as well as the legend that appears when you hover your mouse over the plot.
+  - **Expressions**: Custom calculated expressions to add to the panel:
+    * **Y Axis Expressions**: Add calculated metrics to your graph. You can use any of the logged metrics as well as configuration values like hyperparameters to calculate custom lines.
+    * **X Axis Expressions**: Rescale the x-axis to use calculated values using custom expressions. Useful variables include\*\*_step\*\* for the default x-axis, and the syntax for referencing summary values is `${summary:value}`
+
+1. Click **Apply**.
```

Comment:
The number is off here. In the preview it shows up as 1, not 3.

Reply:
I rewrote this to put the settings into a subsection to work around brittle lists and help with readability.